vnext/JSI/Desktop/JSI.Desktop.vcxproj

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="16.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
>Project [](start = 1, length = 7)

I wonder why these changes are needed here and not in other projects. This project does not create a DLL. We have several such projects. The most important among them are ReactCommon and Folly.

Reply:
I did all projects that have a package.config and are not tests. I'll remove it form this one then.